ebos/ecltracermodel.hh

```diff
@@ -106,9 +107,10 @@ public:
 
         if (!EWOMS_GET_PARAM(TypeTag, bool, EnableTracerModel)) {
             if (simulator_.gridView().comm().rank() == 0) {
-                std::cout << "Warning: Tracer model is disabled but the deck contains the TRACERS keyword\n"
-                          << "The tracer model must be explictly activated using --enable-tracer-model=true\n"
-                          << std::flush;
+                OpmLog::warning("Warning: Keyword TRACERS has only experimental support, and is hence ignored.\n"
```

Comment:
This can be simplified to:'
```
"Warning: keyword TRACERS"
"The experimental tracer ..."
"To use tracers, ..."'
```
i.e. multiple consecutive C strings `"..."` will be automatically joined.

Reply:
Will fix.